### PR TITLE
Add ability to proxy local http requests to alpha

### PR DESCRIPTION
**Pull request by lars-mortenson**
Proxy

Added tests, another screenshot:

Screen Shot 2020-11-17 at 11 49 47 AM

### CODE REVIEW

File: src/alphaProxy.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const { createServer } = require('http');
+
+const alphaProxy = async (baseConfig, callback) => {
+  createServer((req, response) => {
```
Comment (mdlavin) [inline src/alphaProxy.js]: For security reasons, should this be restricted to localhost only?

**Review by mdlavin — CHANGES_REQUESTED** [review]

File: src/alphaProxy.js (hunk 1 above)
Comment (lars-mortenson) [reply src/alphaProxy.js]: Yes, nice catch.

**Review by mdlavin — DISMISSED** [review]

**Review by xdumaine — DISMISSED** [review]
Great work. Much simpler than I thought it'd be.

**Comment by lars-mortenson** [comment]
Just FYI, tests have been more challenging than I expected, partly because I needed a way for the proxy server to tear down between tests. I'm putting them on hold for the moment to work on the current epic, will pick them up when I'm free.

**Review by swain — APPROVED** [review]

**Review by xdumaine — APPROVED** [review]

**Review by mdlavin — APPROVED** [review]

**Pull request merged**